packages/jupyterlab-lsp/src/transclusions/defaults.ts

```diff
@@ -1,10 +1,12 @@
 import { JupyterFrontEndPlugin } from '@jupyterlab/application';
 import { IPYTHON_RPY2_TRANSCLUSIONS } from './ipython-rpy2';
 import { IPYTHON_SQL_TRANSCLUSIONS } from './ipython-sql';
+import { IPYTHON_BIGQUERY_TRANSCLUSIONS } from './ipython-bigquery';
 import { IPYTHON_TRANSCLUSIONS } from './ipython';
 
 export const DEFAULT_TRANSCLUSIONS: JupyterFrontEndPlugin<void>[] = [
   IPYTHON_RPY2_TRANSCLUSIONS,
   IPYTHON_SQL_TRANSCLUSIONS,
-  IPYTHON_TRANSCLUSIONS
+  IPYTHON_TRANSCLUSIONS,
+  IPYTHON_BIGQUERY_TRANSCLUSIONS
```

Comment:
```suggestion
  IPYTHON_BIGQUERY_TRANSCLUSIONS,
  IPYTHON_TRANSCLUSIONS
```

I think that there was a thing with IPython substitutions including a catch-all regexp that has to be registered as the last one. Not sure if this is stlll the case, but probably safer to keep it in that order.